frontend/packages/console-dynamic-plugin-sdk/scripts/parsers/CodeRefTypeReferenceParser.ts

```diff
@@ -1,29 +1,20 @@
-import * as _ from 'lodash';
 import * as tsj from 'ts-json-schema-generator';
 import * as ts from 'typescript';
-import { ConsoleTypeDeclarations } from '../utils/type-resolver';
 
 /**
  * Parse references to `CodeRef<T>` functions as references to `EncodedCodeRef` object literals.
  */
 export class CodeRefTypeReferenceParser implements tsj.SubNodeParser {
   constructor(
     private readonly typeChecker: ts.TypeChecker,
-    private readonly consoleTypeDeclarations: ConsoleTypeDeclarations,
+    private readonly consoleTypeDeclarations: Record<string, ts.Declaration>,
     private readonly getMainParser: () => tsj.NodeParser,
   ) {}
 
   supportsNode(node: ts.Node) {
-    if (ts.isTypeReferenceNode(node)) {
-      const nodeType = this.typeChecker.getTypeAtLocation(node);
-
-      return (
-        nodeType.aliasSymbol?.name === 'CodeRef' &&
-        _.head(nodeType.aliasSymbol?.declarations) === this.consoleTypeDeclarations.CodeRef
-      );
-    }
-
-    return false;
+    return ts.isTypeReferenceNode(node)
+      ? this.typeChecker.getTypeAtLocation(node).aliasSymbol?.name === 'CodeRef'
```

Comment:
Before, we checked that the `CodeRef` type comes from Console specifically.

Now, we just check that the symbol name is `CodeRef`, allowing integration with `CodeRef` type that comes from `@openshift/dynamic-plugin-sdk` package. (We could parse the symbol declaration from both sources and do reference comparison, but I don't think the added complexity is worth it.)